cloudevents/formats/avro-turbo-format.md

```diff
@@ -0,0 +1,109 @@
+# Avro Turbo Event Format for CloudEvents - Version 1.0.3-wip
+
+## Abstract
+
+The Avro Turbo Format for CloudEvents defines how events are expressed in
+the [Avro 1.9.0 Specification][avro-spec].
+
+This differs from the [Avro format](avro-format.md) in that:
+
+- It is optimized for performance, preferring a more compact representation.
+- It only supports spec version 1.0 (any changes to spec version requires changes to the Avro schema,
+  which changes the fingerprint, breaking compatibility).
+- It does not natively support JSON (JSON can be straight-forwardly serialized 
+  to bytes and this was therefore not considered necessary).
+
+## Table of Contents
+
+1. [Introduction](#1-introduction)
+2. [Attributes](#2-attributes)
+3. [Transport](#3-transport)
+4. [Examples](#4-examples)
+
+## 1. Introduction
+
+[CloudEvents][ce] is a standardized and protocol-agnostic definition of the
+structure and metadata description of events. This specification defines how the
+CloudEvents are to be represented as [Avro 1.9.0][avro-primitives].
+
+The [Attributes](#2-attributes) section describes the naming conventions and
+data type mappings for CloudEvents attributes for use as Avro message
+properties.
+
+This specification does not define an envelope format. The Avro type system's
```

Comment:
I'm unclear on exactly what this means. (Isn't it still a wrapper around the data, surfacing the metadata? Isn't that sort of an envelope?)

Reply:
Should we delete?

Reply:
I'd be happy with deleting it, if that's okay with others.

Reply:
If we do remove it perhaps we should do it from both Avro docs for consistency.

Reply:
action?